### PR TITLE
[Reopen] [UPDATE] Medical crafting

### PR DESCRIPTION
**Pull request by Raze500**
### Adds more recipes for the medical category

- Adds a crafteable Large and grenade beaker
- Adds a crafteable syringes and a syringe gun

### What the grenade nerf is?

Adds a crafteable grenade beaker (50u) in to the **medical category** (bcuz muh beakers)
Removes the hability to use normal beakers and large beakers in grenade casins (Water bottles too)

**The craft uses 2 glass and 2 metal sheets, i would add it to the autholate but it would fuck other's people PR's....**

**THIS DONT AFFECT THE PREMADES GRENADES. ALREADY TESTED.**

> Has a chemist main this hurts.

### CODE REVIEW

**Comment by imw-Oculus** [comment]
I'm not sure why this was closed in the first place. @UndeadNappist Can you explain?

**Comment by UndeadNappist** [comment]
I am willing to merge the PR and explain exactly why I closed it once the grenade change is implemented.

I do not actually have a particular interest in who makes that particular change.

**Comment by Raze500** [comment]
Oops some error by byond.

**Comment by Raze500** [comment]
Bruh if you want nerf grenades do it yourself. i will not nerf it beacuse it will waste my time.

**Comment by Raze500** [comment]
Or give good reasons to change my mind to why to do it.

**Comment by Raze500** [comment]
**Okay fuck you adam for put me on reason, damm nerd.**

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -754,7 +754,49 @@
...
+
+/datum/crafting_recipe/solids
+	name = "Solid beaker"
+	result = /obj/item/reagent_containers/glass/beaker/solids
+	reqs = list(/obj/item/stack/sheet/glass = 2,
+				/obj/item/stack/sheet/metal = 2)
+	tools = list(TOOL_WELDER)
+	time = 30
+	category = CAT_MEDICAL
+
```
Comment (UndeadNappist) [inline code/modules/crafting/recipes.dm]: Don't forget to add the recipe to the autolathe datum under security.

File: code/game/objects/items/grenades/chem_grenade.dm [hunk 2]
```diff
@@ -11,7 +11,7 @@
 	force = 2
 	var/stage = EMPTY
 	var/list/beakers = list()
-	var/list/allowed_containers = list(/obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/glass/bottle)
+	var/list/allowed_containers = list(/obj/item/reagent_containers/glass/beaker/solids)
```
Comment (UndeadNappist) [inline code/game/objects/items/grenades/chem_grenade.dm]: You can leave bottles on the list, that's fine - they're 30u after all.

Comment (Raze500) [reply code/game/objects/items/grenades/chem_grenade.dm]: Exist large bottles, that works like a large beaker nappist, that's why i removed it

**Comment by Raze500** [comment]
### It just works

File: code/game/objects/items/grenades/chem_grenade.dm (hunk 2 above)
Comment (UndeadNappist) [reply code/game/objects/items/grenades/chem_grenade.dm]: they're glass/beaker/waterbottle, not /glass/bottle/waterbottle ; this means that waterbottles are actually a child of beakers, not a child of bottles, and are therefore a type of beaker according to the code.

**Pull request merged**